QuestionBot/QuestionBot.UnitTests/Model/MessageEmitterTests.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using NUnit.Framework;
+using QuestionBot.Model;
+using Moq;
+
+namespace QuestionBot.UnitTests.Model {
```

Comment:
IMO, the two tests here do not currently cover all the features of the module. The notion of [code coverage](https://en.wikipedia.org/wiki/Code_coverage) might help here. Basically, to achieve 100% code coverage, all of the lines of code must be executed at least once during the test suite run. However, achieving 100% code coverage does not necessarily mean your module has good tests. Additionally, sometimes it's just not worth it to get 100% code coverage. But it's a good guideline on deciding what to test.